eth-pm/src/App.tsx

```diff
@@ -2,8 +2,9 @@ import "@ethersproject/shims";
 
 import React, { useEffect, useState } from "react";
 import "./App.css";
-import type { WakuPrivacy } from "@waku/interfaces";
-import { AsymDecoder, SymDecoder } from "@waku/message-encryption";
+import type { RelayNode, IDecoder } from "@waku/interfaces";
+import { createDecoder as createSymmetricDecoder } from "@waku/message-encryption/symmetric";
```

Comment:
This is not ideal but I believe it is more of a code smell in the example as decoders for various types of messages are instantiated in the same file.

A better architecture would separate concerns between the various functionalities.

The main issue I believe is the access to the Waku node instance.
Once cleaned up, then this instance can be passed to various component and each component can:
- setup their encoder/decoder
- setup the right hooks on the waku instance (e.g. addObserver)